rpm-readme.md

```diff
@@ -56,19 +56,39 @@ Done.
 
 ```
 
-#### Installing rpm package
+> You may want to check RPM package content with `sudo rpm -ql foglamp-gui-x.y.z-1.x86_64.rpm`
 
-Use the ``rpm`` command
+
+### Installing rpm package
+
+```
+$ sudo cp packages/build/foglamp-gui-x.y.z/RPMS/x86_64/foglamp-gui-x.y.z-1.x86_64.rpm /var/cache/yum/x86_64/.
+```
+
+**Use the `rpm` or `yum localinstall` command**
+
+```
+$ sudo rpm  -i --force /var/cache/yum/x86_64/foglamp-gui-x.y.z-1.x86_64.rpm
+```
+
+> You can also use `--replacefiles` instead of `--force`
+
+OR
 
 ```
-$sudo cp packages/build/foglamp-gui-x.y.z/RPMS/x86_64/foglamp-gui-x.y.z-1.x86_64.rpm /var/cache/yum/x86_64/.
-$sudo rpm  -i --force /var/cache/yum/x86_64/foglamp-gui-x.y.z.x86_64.rpm
+$ sudo yum localinstall /var/cache/yum/x86_64/foglamp-gui-x.y.z-1.x86_64.rpm
```

Comment:
`--force` is not working with `localinstall`, then what should be used instead of `--force` in case of conflicting?

Reply:
Need to figure it out. should I add notes about it as TODO/ FIXME:? And request to use `sudo rpm  -i --force ...` until we fix?

Reply:
Yes, we should add it as TODO.